services/case_filters/case_filter_base.py

```diff
@@ -0,0 +1,117 @@
+import logging
+from abc import abstractmethod
+from typing import List
+
+from models.blaise.blaise_case_information_model import BlaiseCaseInformationModel
+from models.totalmobile.totalmobile_world_model import TotalmobileWorldModel
+
+
+class CaseFilterBase:
+    @property
+    @abstractmethod
+    def wave_number(self) -> int:
+        pass
+
+    @abstractmethod
+    def case_is_eligible(self, case: BlaiseCaseInformationModel) -> bool:
+        pass
+
+    @staticmethod
+    def case_is_part_of_wave(
+        wave_number: int, case: BlaiseCaseInformationModel
+    ) -> bool:
+        if case.wave == wave_number:
```

Comment:
This method could just be `return case.wave == wave_number`

Reply:
https://github.com/ONSdigital/blaise-totalmobile-services/pull/194